### PR TITLE
feat: add Portuguese (pt-BR) translations and update i18n configuration to include the new language

### PR DESCRIPTION
**Pull request by glaucia86**
## Does this introduce a breaking change?

When developers merge from main and run the server, azd up, or azd deploy, will this produce an error?
If you're not sure, try it out on an old environment.

```
[ ] Yes
[x] No
```

## Does this require changes to learn.microsoft.com docs?

This repository is referenced by [this tutorial](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-template)
which includes deployment, settings and usage instructions. If text or screenshot need to change in the tutorial,
check the box below and notify the tutorial author. A Microsoft employee can do this for you if you're an external contributor.

```
[ ] Yes
[x] No
```

## Type of change

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Code quality checklist

See [CONTRIBUTING.md](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/CONTRIBUTING.md#submit-pr) for more details.

- [x] The current tests all pass (`python -m pytest`).
- [ ] I added tests that prove my fix is effective or that my feature works
- [x] I ran `python -m pytest --cov` to verify 100% coverage of added lines
- [x] I ran `python -m mypy` to check for type errors
- [x] I either used the pre-commit hooks or ran `ruff` and `black` manually on my code.

### CODE REVIEW

File: app/frontend/src/i18n/config.ts [hunk 1]
```diff
@@ -24,6 +25,10 @@ export const supportedLngs: { [key: string]: { name: string; locale: string } }
     ja: {
         name: "日本語",
         locale: "ja-JP"
+    },
+    ptBR: {
+        name: "Português",
+        locale: "pt-BR"
```
Comment (pamelafox) [inline app/frontend/src/i18n/config.ts:31]: Thank you Glaucia! If someone comes in with pt-PT as the language code, would they just get English translations then? Is that how it's normally done? I remember learning that the pronunciation of pt-PT is very different from pt-BR in my portuguese class, but I don't know how different the words are, so I always wondered how that was handled for i18n.

Comment (glaucia86) [reply app/frontend/src/i18n/config.ts:31]: > Thank you Glaucia! If someone comes in with pt-PT as the language code, would they just get English translations then? Is that how it's normally done? I remember learning that the pronunciation of pt-PT is very different from pt-BR in my portuguese class, but I don't know how different the words are, so I always wondered how that was handled for i18n.

Yes, there are many differences. But I've heard that our Portuguese cousins prefer to learn Brazilian Portuguese. But if it comes up, let me know and I'll help you out

Comment (glaucia86) [reply app/frontend/src/i18n/config.ts:31]: Let me include here:

```json
 name: "Português Brasileiro",
```

Comment (glaucia86) [reply app/frontend/src/i18n/config.ts:31]: Done! I made the change

**Review by pamelafox — APPROVED** [review]

**Review by pamelafox — APPROVED** [review]

**Pull request merged**